pkg/controller/secret/secret_controller.go

```diff
@@ -465,12 +479,13 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 	reqLogger := log.WithValues("Request.Name", request.Name)
 	reqLogger.Info("Reconciling Secret")
 
-	// This operator is only interested in the 3 secrets listed below. Skip reconciling for all other secrets.
+	// This operator is only interested in the 3 secrets & 1 configMap listed below. Skip reconciling for all other objects.
 	// TODO: Filter these with a predicate instead
 	switch request.Name {
 	case secretNamePD:
 	case secretNameDMS:
 	case secretNameAlertmanager:
+	case cmNameManagedNamespaces:
```

Comment:
Now that there are multiple types being handled here, a type check, at least for the once case, should be added.

Reply:
Looks like `request` is a `reconcile.Request` object, which only contains a name & namespace, so I don't know if a type check is feasible. 

The good news is that even if there is both a secret and configmap named after `cmNameManagedNamespaces`, the reconcile loop should still ignore the secret and only interact with the configmap, unless I'm misunderstanding the intention here.

Reply:
Ignore might be the wrong word, but I don't think it would have any ill-effect. 

In a case where a configmap name matches a CAMO secret name and triggers a reconcile (or vice-versa), the request name is only used to lookup the correct objects and act on them. In this case, the reconcile _is_ called unnecessary, but it shouldn't result in a different `alertmanager-main` secret, since the same objects are used to generate it each time.